packages/amplify-category-storage/resources/generateSchemas.ts

```diff
@@ -0,0 +1,82 @@
+import { getProgramFromFiles, buildGenerator, PartialArgs } from 'typescript-json-schema';
```

Comment:
is there a reason we need to duplicate this file from `amplify-headless-interface`?